packages/notebook/src/browser/contributions/notebook-cell-actions-contribution.ts

```diff
@@ -26,35 +26,42 @@ import { NotebookCellOutputModel } from '../view-model/notebook-cell-output-mode
 import { CellEditType } from '../../common';
 
 export namespace NotebookCellCommands {
+    // Parameters: notebookModel: NotebookModel | undefined, cell: NotebookCellModel
```

Comment:
Suggestion: It would probably be better if we make this into JSDoc, so TypeScript's tooling makes it easier to see.